packages/kestrel_core/src/kestrel/ir/graph.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 import json
+import copy
```

Comment:
We should try to keep the import sorted according to PEP 8.  A simple way to do this is using the `isort` tool (I find i works best *after* `unimport` but *before* `black`).

Reply:
Thanks for the suggestion, will add to the github action.